src/bg/on-user-script-notification.js

```diff
@@ -40,15 +40,19 @@ chrome.runtime.onConnect.addListener(onUserScriptNotification);
 
 chrome.notifications.onClicked.addListener(id => {
   let port = portMap.get(id);
-  port.postMessage({type: 'onclick'});
+  if (port) {
+    port.postMessage({type: 'onclick'});
+  }
 });
 
 
 chrome.notifications.onClosed.addListener(id => {
   let port = portMap.get(id);
-  portMap.delete(id);
-  port.postMessage({type: 'ondone'});
-  port.disconnect();
+  if (port) {
```

Comment:
Rather than indenting ~the entire method, I'd prefer `if (!port) return;` as a general pattern.